src/styles/index.scss

```diff
@@ -1,3 +1,129 @@
-body {
+.page {
   margin: 0;
+  box-sizing: border-box;
```

Comment:
The `box-sizing` property should be set globally for all elements using the universal selector `*`, and not just for the `.page` class, to ensure consistency across the entire project.